content/software-triage.md

```diff
@@ -46,44 +46,8 @@ Do the symptoms go away if the system is suspended, rebooted or shut down and th
     E.g. windows frozen in place, windows will not resize. Stuck in full-screen mode, etc.
     Mouse freezes intermittently or stutters it moves across the screen, or keyboard input lags.
 
-## Causes
```

Comment:
These are not negative words, they are descriptive words. And they are all potential causes of software issues.

The software is corrupted because the code was written correctly but something happened to it in download or install.

Old software is out of date.

If the wrong driver or driver version is installed, its an incorrect version.